exercises/ruby-1-b/ruby_1_b_example.rb

```diff
@@ -1,12 +1,23 @@
-VOWELS = %w{a e i o u}
-def word_with_most_vowels(sentence)
-  words = sentence.split(" ")
-  word_vowels = words.map do |word|
-    [
-      word,
-      word.downcase.chars.select{|l|VOWELS.include?(l)}.count
-    ]
+class Library
+  def self.books
+    @books ||= import_books
+  end
+
+  def self.import_books
+    file_name = "books.txt"
+    IO.readlines(file_name, chomp: true).drop(1)
+      .map { |line| line.split(";") }
```

Comment:
I prefer single quoted strings when there is no need for string interpolation or special symbols but I know this is a style preference so it's up to you. Do you know if we use Rubocop or something similar?

Reply:
I personally try to use double quotes always (although sometimes some single quotes slip in there). But I'm not very pedantic about this issue, so if there's some project style guide that prefers the single quote style, then I'm fine with changing this

Reply:
As long as they are [consistent](https://rubystyle.guide/#consistent-string-literals) I don't mind either, though I prefer single quotes by default.